src/v/cluster/metrics_reporter.h

```diff
@@ -18,6 +18,7 @@
 #include "features/fwd.h"
 #include "http/client.h"
 #include "model/metadata.h"
+#include "net/unresolved_address.h"
```

Comment:
this is it yeah?

Reply:
oh yeah sorry I should have called it out